### PR TITLE
RHBPMS-4536 - Repeatly triggering Timers in mulitinstance subprocesse…

### PR DESCRIPTION
**Pull request by mswiderski**
…s with runtime strategy PER_PROCESS_INSTANCE (#703)

(cherry picked from commit 2bfc3b2b9b0b392a35a11b28d9ee3a60c1561fb0)

### CODE REVIEW

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

File: jbpm-services/jbpm-kie-services/src/test/resources/multi-remotable/pojo-pom.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+ <groupId>org.test</groupId>
+ <artifactId>jbpm-kie-services-filter-test-pojo</artifactId>
+ <version>1.0.0-SNAPSHOT</version>
+
+ <dependencies>
+  <dependency>
+   <groupId>org.kie</groupId>
+   <artifactId>kie-api</artifactId>
+   <version>6.4.0.Final</version>
```
Comment (krisv) [inline jbpm-services/jbpm-kie-services/src/test/resources/multi-remotable/pojo-pom.xml]: Are we hardcoding this on purpose?

Comment (mswiderski) [inline jbpm-services/jbpm-kie-services/src/test/resources/multi-remotable/pojo-pom.xml]: we need to reference a version of kie-api as it uses @Remotable annotation. I guess we could change it to same version as project just wondering if we use 6.5.1-SNAPSHOT will that be automatically replaced with next version on release time, @psiroky @mbiarnes wdyt?

Comment (psiroky) [inline jbpm-services/jbpm-kie-services/src/test/resources/multi-remotable/pojo-pom.xml]: I believe we should use `${version.org.kie}` and setup maven-resource-plugin to filter this file (and thus replace the placeholder with actual version). If we hardcode `6.5.1-SNAPSHOT` it won't get automatically replaced.

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by krisv** [comment]
+1

**Comment by mswiderski** [comment]
merged manually

**Pull request closed without merging**